spec/system/support/login_helpers.rb

```diff
@@ -1,5 +1,6 @@
 module LoginHelpers
   def login_as_a_provider
+    allow(PDA::ProviderDetails).to receive(:call).and_return(true) # this stubs out calls to the pda schedules endpoint
```

Comment:
I have added this to stub out calls to the PDA schedules endpoint and resolve a number of failing specs that use this helper - not sure if that is the best approach?